CONTRIBUTING.md

```diff
@@ -19,62 +19,47 @@ GitHub Usage & Branching
 Once you decide you want to contribute to phpDocumentor (which we really appreciate!) you can fork the project from
 http://github.com/phpDocumentor/phpDocumentor2.
 
-Currently phpdocumentor version 3.0 is developed in our develop branch. Large parts of the code in there will be rewritten or 
-removed. If you want to contribute to phpdocumentor v3.0 create your feature branch from develop. If you want to fix a bug in the
-current released version base your branch on master. Please create *aways* a new branch for each feature/bugfix you want to contribute.
+Currently phpdocumentor version 3.0 is developed in our develop branch. Large parts of the code in there will be
+rewritten or removed. If you want to contribute to phpdocumentor v3.0 create your feature branch from develop. If you
+want to fix a bug in the current released version base your branch on master. Please create *aways* a new branch for
```

Comment:
* I'd wrap `develop` and `master` with code ticks, to make them stand out.
* Commas to separate clauses:
  - "to phpdocumentor v3.0, create your"
  - "current release version, base your branch"
* "Please *always* create"  (word order, spelling)

Reply:
Fixed and amended to the current commit